db/schema.rb

```diff
@@ -1,4 +1,18 @@
+# encoding: UTF-8
```

Comment:
@andela-anwocha Please revert this change, you should never make a change directly in your schema, you must create a migration to make any change